shared/schema.ts

```
@@ -2,6 +2,18 @@ import { pgTable, text, serial, integer, boolean, timestamp, jsonb, varchar } fr
 import { relations } from "drizzle-orm";
 import { createInsertSchema } from "drizzle-zod";
 import { z } from "zod";
+import { integer, pgTable, varchar } from "drizzle-orm/pg-core";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'integer'. Consider to delete it or rename it.

'integer' is defined here:


(lint/suspicious/noRedeclare)